public/src/pages/marketplace/Partners/Partners.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { FC, useEffect, useState } from 'react';
+import { DOMParser } from 'xmldom';
+
+import OwlCarousel from 'react-owl-carousel';
+import 'owl.carousel/dist/assets/owl.carousel.css';
+import 'owl.carousel/dist/assets/owl.theme.default.css';
+
+import { searchPartners, partnerLogin } from '../../../api/httpClient';
+import { Partner } from '../../../interfaces/Partner';
+
+export const Partners: FC = () => {
+  const PARTNER_DEFAULT_USERNAME = 'walter100';
+  const PARTNER_DEFAULT_PASSWORD = 'Heisenberg123';
+
+  const [partners, setPartners] = useState<Array<Partner>>([]);
+
+  const partnerNameSearchEP = () => {
```

Comment:
`fetchPartners` would be a better name as `partnerNameSearchEP` is misleading, this function is not an endpoint :)